DeleteMiddleNode.py

```diff
@@ -0,0 +1,36 @@
+class ListNode:
+    def __init__(self, value=0, next=None):
+        self.value = value
+        self.next = next
+
+def deleteNode(node):
```

Comment:
This doesn't look right to me. You should be finding and deleting the middle node in this function, not just deleting whatever node is the incoming argument.